build.gradle

```diff
@@ -106,6 +107,21 @@ task showDifferencesStrict(type: JavaExec) {
     classpath = sourceSets.test.runtimeClasspath
     args('test-data/oh.txt-result-strict', 'test-data/oh.txt-result-strict-temp', 'test-data/oh.txt', 'test-data/diff-strict.txt')
 }
+
+task individualTesting(type: JavaExec) {
+    main = "ch.poole.openinghoursparser.IndividualTest"
+    classpath = sourceSets.test.runtimeClasspath
+    standardInput = System.in
+    args('false')
+}
+
+task individualTestingStrict(type: JavaExec) {
+    main = "ch.poole.openinghoursparser.IndividualTest"
+    classpath = sourceSets.test.runtimeClasspath
+    standardInput = System.in
+    args('true')
+}
+

```

Comment:
Both tasks should be moved below line 126 as 125 and 128 belong to the showDifferencesStrict task